english-helper/src/features/user/LoginPage.tsx

```diff
@@ -23,8 +35,9 @@ export default function LoginPage() {
         .matches(/[@$!%*?&]/, "must contain a special character")
         .required(" is required"),
     }),
-    onSubmit: (values) => {
-      alert(JSON.stringify(values, null, 2));
+    onSubmit: async ({ email, password, remember }) => {
+      logIn({ email, password, dispatch });
+      alert(JSON.stringify({ email, password, remember }, null, 2));
     },
   });
 
```

Comment:
const emailValidation = !!(formik.touched.email && formik.errors.email);
const passwordValidation = !!(formik.touched.password && formik.errors.password);
simplify this validation